src/rgw/setup/radosgw_start

```diff
@@ -2,15 +2,13 @@
 
 INDEX=$1
 CONFIG_FILE=$2
-LOG_FILE=$3
-RGW_CWD=$4
+RGW_CWD=$3
 
 [ -z "$RGW_CWD" ] && RGW_CWD="$PWD"
 
 echo "Change RGW CWD to dir $RGW_CWD, to retain core files in there"
 cd $RGW_CWD
 
 set -o pipefail;
-/usr/bin/radosgw -f --name client.rgw-$INDEX -c $CONFIG_FILE --no-mon-config 2>&1 \
- | while read line; do printf "%s %s\n" "$(date +'[%Y-%m-%d %H:%M:%S,%03N]')" "$line"; done | tee $LOG_FILE
```

Comment:
timestamp per motr log would still be needed, while dumping on console. So i believe we can just avoid tee-ing it to additional $LOG_FILE

Reply:
wait, so direct libMotr messages on console wont have timestamp, while rgw redirected logs would have timestamp already, so with timestamp addition here, rgw logs would have double timestamp ? Can you cross check this with current RGW container logs

Reply:
@cdeshmukh  yes. this is correct..
In rgw_startup.log file, 
a) for rgw logs, it has two timestamps one is appended by this script & another was its default.  
b) for libmotr logs, it has only one timestamp.
 
![image](https://user-images.githubusercontent.com/65209830/187115209-e55e2d86-df75-4d7f-b46b-9e4573d5755d.png)

Reply:
Would be better if libMotr itself adds a timestamp in their code
@nkommuri it doesnt make sense for Motr client like RGW to add timestamps excplicitly for libMotr generated console logs, what your take on this ?

Reply:
@madhavemuri @nkommuri we will remove timestamp prepend for libMotr console logs, due to complexities mentioned above, and its not right way of adding log timestamps also. Better libMotr code does it by itself

Reply:
Sure